From 5a554e0b34810cc769457eecb00577b9bfbca177 Mon Sep 17 00:00:00 2001
From: "ray.lue" <ray.lue@mic.com.tw>
Date: Wed, 10 Apr 2019 14:05:57 +0800
Subject: [PATCH] force scanned fru as ID0 Note: the Intel set FRU ID0
 dedicated for built-in basebaord fru image,all the other scanned FRU's ID
 will be hashed. If there is no baseboard fru image, the SDR commands can't
 response correctly. this modificaiton walk aroud this issue.

---
 src/storagecommands.cpp | 3 +++
 1 file changed, 3 insertions(+)
 mode change 100644 => 100755 src/storagecommands.cpp

diff --git a/src/storagecommands.cpp b/src/storagecommands.cpp
old mode 100644
new mode 100755
index 39405c5..8c80037
--- a/src/storagecommands.cpp
+++ b/src/storagecommands.cpp
@@ -214,6 +214,9 @@ ipmi_ret_t replaceCacheFru(uint8_t devId)
         }
         std::pair<uint8_t, uint8_t> newDev(fruBus, fruAddr);
 
+		//ray__ debug force hash = 0
+		fruHash = 0;
+
         bool emplacePassed = false;
         while (!emplacePassed)
         {
